apps/browser/src/components/googleAnalytics.tsx

```diff
@@ -1,22 +1,39 @@
 interface Window {
-    gtag: any;
+  gtag: any;
+}
+declare var window: Window;
+
+export const trackMessageSent = (message: string) => {
+  try {
+    if (window.gtag) {
+      // Check if gtag is initialized
+      window.gtag("event", "message_sent", {
+        event_category: "Chat",
+        event_label: message, // Log the message as a label
+        value: 1, // 1 message was sent
+      });
+      console.log("GA event sent");
```

Comment:
No need to have superfluous logging to the console, don't think this is useful. If you agree, let's remove here and below in the other function.

Reply:
NOTE: logging of errors is helpful.